LibraryApp/src/app/screens/BookDetails/components/CommentsList/index.js

```diff
@@ -0,0 +1,36 @@
+import React, { Component } from 'react';
+import { View, FlatList } from 'react-native';
+import PropTypes from 'prop-types';
+
+import styles from './styles';
+import Comment from './components/Comment';
+
+class CommentsList extends Component {
+  renderItem = ({ item }) => {
+    return <Comment item={item} />;
+  };
+
+  keyExtractor = ({ id }) => id;
+
+  render() {
+    const { comments } = this.props;
+    return (
+      <View style={styles.container}>
```

Comment:
I think you could add this style inside `FlatList`

Reply:
yes i can! :D